Because CBFlib_0.9.7_Data_Files_Input.tar.gz contains a single file
(._CBFlib_0.9.7_Data_Files_Input) outside the
CBFlib_0.9.7_Data_Files_Input directory, conda will not move the
directory contents up one level and this patch must provide paths.
--- data-in/CBFlib_0.9.7_Data_Files_Input/9ins.cif
+++ data-in/CBFlib_0.9.7_Data_Files_Input/9ins.cif
@@ -1,5 +1,5 @@
-###CBF: VERSION 1.7.10
-# CIF file written by CBFlib v0.9.5
+###CBF: VERSION 1.7.11
+# CIF file written by CBFlib v0.9.7
 
 data_9INS
 
--- data-in/CBFlib_0.9.7_Data_Files_Input/testflatin.cbf
+++ data-in/CBFlib_0.9.7_Data_Files_Input/testflatin.cbf
@@ -1,5 +1,5 @@
 ###CBF: VERSION 1.7.11
-# CBF file written by CBFlib v0.9.6
+# CBF file written by CBFlib v0.9.7
 
 data_testflat
 
--- data-in/CBFlib_0.9.7_Data_Files_Input/testflatpackedin.cbf
+++ data-in/CBFlib_0.9.7_Data_Files_Input/testflatpackedin.cbf
@@ -1,5 +1,5 @@
 ###CBF: VERSION 1.7.11
-# CBF file written by CBFlib v0.9.6
+# CBF file written by CBFlib v0.9.7
 
 data_testflat
 
--- data-in/CBFlib_0.9.7_Data_Files_Input/testrealin.cbf
+++ data-in/CBFlib_0.9.7_Data_Files_Input/testrealin.cbf
@@ -1,5 +1,5 @@
 ###CBF: VERSION 1.7.11
-# CBF file written by CBFlib v0.9.6
+# CBF file written by CBFlib v0.9.7
 
 data_testreals
 
